includes/api/class-edd-api.php

```diff
@@ -101,6 +137,15 @@ class EDD_API {
 	 * @since 1.5
 	 */
 	public function __construct() {
+
+		$this->versions = array(
+			'v1' => 'EDD_API_V1',
```

Comment:
This is the map from URL to class, so the `v` is correct since you'r do /edd/v1/, not /edd/1/